src/main/java/org/elasticsearch/common/jna/Kernel32Library.java

```diff
@@ -35,6 +37,27 @@
  */
 public class Kernel32Library {
 
+    public static final int PROCESS_ALL_ACCESS = 0x0501;
+    public static final int PAGE_NOACCESS = 0x0001;
+    public static final int PAGE_GUARD = 0x0100;
+
+    public static final int MEM_COMMIT = 0x1000;
+
+    public static class MEMORY_BASIC_INFORMATION extends Structure {
```

Comment:
No, doesn't need to be.  It was just a convention to emphasize that it represents a native structure, but it can and probably should be camel cased instead.